_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -222,16 +221,39 @@ class HealthCareApp extends React.Component {
       case 'financial-assessment':
         return {
           receivesVaPension: this.state.applicationData['personal-information']['va-information'].receivesVaPension,
-          neverMarried: this.state.applicationData['personal-information']['name-and-general-information'].maritalStatus === '' ||
-            this.state.applicationData['personal-information']['name-and-general-information'].maritalStatus === 'Never Married'
+          neverMarried: this.state.applicationData['personal-information']['name-and-general-information'].maritalStatus === '' || this.state.applicationData['personal-information']['name-and-general-information'].maritalStatus === 'Never Married'
         };
       default:
         return undefined;
     }
   }
 
+  // Make sure the number of children reported have corresponding rows in the annual-income section.
+  // TODO: This should really be looking to match the shortname of instead of just
+  //       just assuming the index between child info and child income match.
+  updateChildrenIncome(children) {
+    const childIncome = this.state.applicationData['financial-assessment']['annual-income'];
```

Comment:
`this.state.applicationData` no longer exists, it is now `blankVeteran` in `_health-care/_js/reducers/veteran/index.js`. Since this function updates our store, I think we would want to create an action and reducer for this. The thing I'm not sure about is when this action should be dispatched. Currently we're calling this function whenever a change is made and when the path equals `/financial-assessment/child-information`, so we could match that and dispatch the action when a change is made in `ChildInformationSection` (since we want to add child income rows when a child is added). The function itself will change slightly too, based on how the other reducer functions are structured in creating a new state object and assigning it new values.